org_fedora_oscap/data_fetch.py

```diff
@@ -196,3 +196,11 @@ def _curl_fetch(url, out_file, ca_certs=None):
         else:
             msg = "Failed to fetch data: %s" % err
             raise FetchError(msg)
+
+    if protocol in ("http", "https"):
+        return_code = curl.getinfo(pycurl.HTTP_CODE)
+        if 400 <= return_code < 600:
+            msg = (
+                "Failed to fetch data - the request returned HTTP error code {return_code}"
```

Comment:
Missing `f`

Reply:
Sorry, this is .format, disregard my comment, please

Reply:
It's probably better to move to f, RHEL8 allows that and it removes that annoying boilerplate.